.one-pipeline.yaml

```diff
@@ -191,8 +191,8 @@ containerize:
      DIGEST="$(skopeo inspect docker://$IMAGE | grep Digest | grep -o 'sha[^\"]*')"
      #ARCH="$(echo $i | grep -o '\(amd64\|s390x\|ppc64le\)$' || echo 'amd64')"
      ARCH=amd64
-     echo "Saving artifact catalog-$i name=$IMAGE digest=$DIGEST manifest=false"
-     save_artifact catalog-$i type=image name="$IMAGE" "digest=$DIGEST" "arch=$ARCH" "manifest=false"
+     echo "Saving artifact catalog-$i name=$IMAGE digest=$DIGEST"
+     save_artifact catalog-$i type=image name="$IMAGE" "digest=$DIGEST" "arch=$ARCH" 
```

Comment:
why is type hard coded to image here?

Reply:
The catalog currently does not have a manifest image.  It will in the future, but there are other changes that will be required at that time to this section of code.